test/TEST-38-FREEZER/testsuite.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
```

Comment:
Also, to make the debugging much easier (especially now with the `busctl` timeout), it may be worthwhile to enable debug logging and log it directly to the console, like we do in other tests:

https://github.com/systemd/systemd/blob/b1ccd384d6d3524484f0153ab23eda8ad67b476a/test/TEST-23-TYPE-EXEC/testsuite.sh#L5-L6

Reply:
Fixed